config/kloubak2-subt-estop-lora-jetson.json

```diff
@@ -214,6 +214,14 @@
           "init": {
             "min_observations": 2
           }
+      },
+      "fromros": {
```

Comment:
note, that this change breaks "autodetection" of robot K2 vs Skiddy in mapping server, but I actually prefer unified streams names, i.e. keep it, thanks :)